source/_components/tesla.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "Tesla"
+description: "Instructions for how to integrate Tesla car into Home Assistant."
+date: 2017-08-28 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Hub
+ha_release: 0.53
+ha_iot_class: "Cloud Polling"
+---
+
+
+The `Tesla` platform offers integration with the [Tesla](https://auth.tesla.com/login) cloud service and offers presence detection as well as sensors such as charger state and temperature.
+
+This platform provides next components:
```

Comment:
**Spelling & Grammar**:
- Suggestion: `This platform provides next components:` -> `This platform provides the following components:`